src/main/java/edu/greenblitz/pegasus/subsystems/Indexing.java

```diff
@@ -0,0 +1,72 @@
+package edu.greenblitz.pegasus.subsystems;
+
+import com.revrobotics.ColorSensorV3;
+import edu.greenblitz.GBLib.src.main.java.edu.greenblitz.gblib.subsystems.GBSubsystem;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import java.awt.*;
+
+
+public class Indexing extends GBSubsystem {
+	/**
+	 * This class is in-charge of checking if an enemy ball is trying to get to the funnel (and eventually to the shooter)
+	 * by using color sensor
+	 *
+	 * @see ColorSensorV3 colorSensorV3
+	 */
+
+	private static Indexing instance;
+	private ColorSensorV3 cs;
```

Comment:
![image](https://github.com/user-attachments/assets/635a8012-ba37-4064-8ef8-77e859469ecb)
C#? Czech language? Computer science? It has a name